linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/rule.yml

```diff
@@ -77,3 +77,20 @@ ocil: |-
     ciphers are in use, run the following command:
     <pre>$ sudo grep Ciphers /etc/ssh/sshd_config</pre>
     The output should contain only those ciphers which are FIPS-approved.
+
+warnings:
+    - general: |-
+        The system needs to be rebooted for these changes to take effect.
+    - regulatory: |-
+        The Federal Information Systems Modernization Act (FISMA), requires cryptography protecting sensitive
+        or valuable data to undergo FIPS 140-2 validation. The U.S. National Institute of Standards and
+        Technology (NIST) views unvalidated cryptography as providing no protection to the information or
+        data—in effect the data would be considered unprotected plaintext. If the agency specifies that the
+        information or data be cryptographically protected, FIPS 140-2 is applicable. This configuration
+        check will fail on platforms lacking FIPS 140-2 validation, such as the CentOS, Scientific Linux,
+        and Fedora projects even if FIPS 140-2 ciphers can be installed and enabled.
+        <br /><br />
+        See {{{ weblink(link="https://csrc.nist.gov/Projects/cryptographic-module-validation-program") }}}</b>
+        for more information about the Cryptographic Validation Program.
+        A list of FIPS certified vendors and products can be found at
```

Comment:
```suggestion
        A list of FIPS validated cryptographic modules can be found at
```

Reply:
@shawndwells let's keep the original language here... people understand certified vendors and products especially those not familiar with NIST and FIPS which there are quite a few.